docker-compose.production.yml

```diff
@@ -116,6 +116,27 @@ services:
       options:
         max-size: "512m"
         max-file: "4"
+  importbot:
+    image: oldev:latest
+    build:
+      context: .
+      dockerfile: docker/Dockerfile.oldev
+    command: docker/ol-importbot-start.sh
+    restart: always
+    hostname: "$HOSTNAME"
+    environment:
+      - OL_CONFIG=conf/openlibrary.yml
+      - PYENV_VERSION=${PYENV_VERSION:-}
+    expose:
+      - 7100
+    depends_on:
+      - db
```

Comment:
Not entirely sure if it does; can drop for now (helps avoid errors, otherwise we'll always need `--no-deps`)
```suggestion
```